src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -266,6 +303,13 @@ export const DownloadConfirmation: React.FunctionComponent<DownloadConfirmationP
         return <></>
     }
   }
+  if (showDownloadList && SynapseClient.isInSynapseExperimentalMode()) {
+    // go to the Download Cart Page
+    if (downloadCartPageUrl)
+      window.location.href = downloadCartPageUrl
```

Comment:
For the first time, I'm going to have a route available to all portals (they do not need to define it in the config)